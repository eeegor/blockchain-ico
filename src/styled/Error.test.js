import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Error } from '.';

describe('Error', () => {
	test('renders default error', () => {
		const tree = renderer.create(<Error>Title</Error>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
