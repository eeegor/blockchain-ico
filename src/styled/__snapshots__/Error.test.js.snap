// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders default error 1`] = `
.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
>
  Title
</div>
`;
